.travis.yml

```diff
@@ -30,6 +30,7 @@ branches:
     only:
         - master
         - /^\d.\d+$/
+        - ezp-26430-js_translator_dumper-improvements
```

Comment:
This commit will not be merged, it's just because this PR is not based on master but on a custom branch.